js/arena.js

```diff
@@ -0,0 +1,35 @@
+ 'strict mode'
+ 
+ export default class Arena {
+     constructor(name , gladiators = []) {
+
+         this.name = name.charAt(0).toUpperCase() + name.slice(1).toLowerCase();
+         this.gladiators = []
+     }
+     addGladiator(gladiator) {
+         if (this.gladiators.length < 2) {
+            this.gladiators.push(gladiator);
+     };}
+
+ 
+     
+
+     fight() {
+         if (this.gladiators.length == 2) {
+             let firstFighter = this.gladiators[0].weapon
+             let secondFighter = this.gladiators[1].weapon
+             if (firstFighter == secondFighter) return console.log("Both eliminated")
```

Comment:
with both eliminated, you need to remove gladiators from the Arena. use `this.gladiators.pop()` twice.